tests/benchmark/custom_memory_manager/include/platform.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#if defined(_WIN32) && defined(_MSC_VER)
+#define WIN_MSVC
```

Comment:
Should we give these a more specific name such as WIN_MSVC_BENCHMARK

Reply:
Not a bad idea. Will fix it.